projects/plugins/jetpack/extensions/blocks/sharing-buttons/style.scss

```diff
@@ -28,6 +28,11 @@
 	}
 }
 
+.editor-styles-wrapper .wp-block-jetpack-sharing-buttons {
+	gap: 0;
+	padding-left: 0;
```

Comment:
`padding-inline-start` would be more internationalization-friendly.

Reply:
Thanks for the suggestion! Updated.